web/pages/fast.svelte

```diff
@@ -8,15 +8,26 @@
     openedFastPost,
     fastMessages as t
   } from '@slowreader/core'
+  import { onMount } from 'svelte'
 
   import { getURL } from '../stores/router.js'
   import Button from '../ui/button.svelte'
   import Loader from '../ui/loader.svelte'
   import PostCard from '../ui/post-card.svelte'
   import TwoStepsPage from '../ui/two-steps-page.svelte'
+
+  let twoStepsPageInstance: TwoStepsPage
+
+  onMount(() =>
+    fastPosts.subscribe(() => {
```

Comment:
```suggestion
    return fastPosts.subscribe(() => {
```

Note, that in [PR check-list](https://github.com/hplush/slowreader/blob/main/docs/pull_request_template.md) we have an important rule:

>  Don’t rush. Check all changes in PR again.

Reply:
It seems to me that this is identical code, but is my version more compact? Just returning `fastPosts.subscribe` result from callback

Reply:
This is my bad, I missed that you do not have `{` in `onMount(() =>`.

Can we use:

```js
onMount(() => {
  return fastPosts.subscribe(() => {
```

If I read it wrong, somebody else can do it too.